lib/components/Actions/Button/index.stories.tsx

```diff
@@ -19,81 +21,179 @@ const meta = {
     variant: "default",
     onClick: fn(),
     label: "Click me",
+    size: "md",
+  },
+  argTypes: {
+    variant: {
+      control: "select",
+      options: [
+        "default",
+        "critical",
+        "neutral",
+        "ghost",
+        "outline",
+        "promote",
+      ],
+      description: "Visual style variant of the button",
+    },
+    size: {
+      control: "select",
+      options: ["sm", "md", "lg"],
+      description: "Size of the button",
+    },
+    label: {
+      control: "text",
+      description: "Button text content",
+    },
+    icon: {
+      control: "boolean",
+      description: "Whether to show an icon",
+    },
+    disabled: {
+      control: "boolean",
+      description: "Disabled state of the button",
```

Comment:
I am counter such obvious descriptions. It is like useless javadocs in Java:

```
/**
/* sets value
/**
public void setValue(int x) { ... }
```

![image](https://github.com/user-attachments/assets/2f333f80-8895-41d6-8925-56b50c14bc6f)